easylog/benchmark build failed

When build with "cmake --build . --config debug", it failed and this is the log

```
yalantinglibs/src/easylog/benchmark/main.cpp: In function 'void test_glog()':
yalantinglibs/src/easylog/benchmark/main.cpp:62:3: error: 'FLAGS_timestamp_in_logfile_name' was not declared in this scope
   62 |   FLAGS_timestamp_in_logfile_name = false;
      |   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
gmake[2]: *** [src/easylog/benchmark/CMakeFiles/easylog_benchmark.dir/build.make:76: src/easylog/benchmark/CMakeFiles/easylog_benchmark.dir/main.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:1354: src/easylog/benchmark/CMakeFiles/easylog_benchmark.dir/all] Error 2
```
I build on the recommended OS and the default glog version seems no FLAGS_timestamp_in_logfile_name

OS version: Ubuntu 22.04.5 LTS (gcc 11.4.0)
glog version: libgoogle-glog-dev/jammy,now 0.5.0+really0.4.0-2 amd64 [installed]

--- a/src/easylog/benchmark/main.cpp
+++ b/src/easylog/benchmark/main.cpp
@@ -59,7 +59,6 @@ void test_glog() {
   std::filesystem::remove("glog.txt");
   FLAGS_log_dir = ".";
   FLAGS_minloglevel = google::GLOG_INFO;
-  FLAGS_timestamp_in_logfile_name = false;
   google::SetLogDestination(google::INFO, "glog.txt");
   google::InitGoogleLogging("glog");
 

